_projects/2021/sunpy/3dplotting.md

```diff
@@ -0,0 +1,50 @@
+---
+name: 3D plotting
+desc: Provide 3D plotting capabilities to SunPy
+# add a short one line description of your project
+requirements:
+# Student requirements:
+ - Knowledge on pythonic physics.
+ - Familiar with python plotting tooling
+difficulty: medium
+issues:
+# Related issues (if any)  to this project.
+ - https://github.com/sunpy/sunpy/pull/4591
+mentors:
+# First person in contact; mentors may change before project starts.
+# GitHub handles
+ - dstansby
+initiatives:
+ - GSOC
+tags:
+# Different technologies needed
+ - python
+ - plotting
+ - 3D
+collaborating_projects:
+# suborganisation(s) to which this project belongs.
+ - sunpy
+---
+This is an awesome project idea for suborg.
```

Comment:
```suggestion
The SunPy ecosystem currently has well developed support for making 2D plots (primarily with Matplotlib), but currently no interface for making 3D plots.

In this project the student will code new tools for taking common datasets used in the SunPy ecosystem (e.g. images of the Sun, maps of the Sun, magnetic field lines from models of the Sun), and plotting them in 3D. This will initially be done using the pyvista plotting toolkit, but we would like to make the tools general enough that other plotting toolkits (e.g. plotly) can be used too.
```

Reply:
Woops, this was meant to go under description.